scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts

```diff
@@ -0,0 +1,60 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import {
+  CometInterface,
+  Liquidator
+} from '../../build/types';
+
+// XXX this is Kovan-specific
```

Comment:
The token address seems to be DAI on mainnet.

Reply:
I think they've tried to use the same deployer/nonce across networks as much as possible

Reply:
removed this comment